src/main/java/com/codejoust/main/model/problem/ProblemContainer.java

```diff
@@ -0,0 +1,42 @@
+package com.codejoust.main.model.problem;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity // This tells Hibernate to make a table out of this class
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class ProblemContainer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    
+    // The problem associated with this game report
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "problem_table_id")
+    private Problem problem;
```

Comment:
One thing to keep in mind for the future - if deleting a problem, will it automatically make this field be null? Cause IIRC there were some issues in the past with deleting problems because tags were still linked to them, so wondering if the same would be true here or if there's a setting to automatically remove those associations

Reply:
This is a good point - I'm not sure, so this is definitely something I'll test before finalizing this feature. 👍